src/controllers/observation-controller.ts

```diff
@@ -63,29 +72,18 @@ export default class ObservationController extends AbstractController {
       const langOrigin = LocaleOrigin[lang] ? LocaleOrigin[lang] : 'desc_eng';
 
       const paramsSearch = parsePageParams(req.query);
-      const paramsAggregation = parseWhereParams(req.query, req.user);
+      const paramsAggregation = parseWhereParams(req.user, req.query);
       const observations = await this.observations.findAndCount(Object.assign(paramsSearch, paramsAggregation));
```

Comment:
please, use here de-structuring to improve code readability

Reply:
I'd asked for this change: c3e4aae 
@SerafimovichEugene

Reply:
now I see))